classroom.rb

```diff
@@ -0,0 +1,13 @@
+# defines the Classroom class
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] When adding a student to a classroom it also sets the classroom for the student:

Just add: 

`student.classroom = self`